### ISSUE TITLE
A11y - Cell toolbar & Markdown toolbar - Enter after selecting Delete or Hide/Show Preview does nothing

### ISSUE THREAD

**Issue by halerankin** [microsoft/azuredatastudio#11235]
- Azure Data Studio Version: 1.23.0

### Test using **Narrator** in **Windows**:
1. Add a new cell.
2. Navigate via keyboard through the cell toolbar icons to the **Delete** icon.
3. Hit Enter.
Result: Nothing happens

4. Navigate from there to the **Move cell up** and hit Enter.
Result: Cell is deleted. _(Cell should have moved up)_

5. Add a new cell.
6. Navigate via keyboard through the cell toolbar icons to the **Move cell down** icon.
7. Hit Enter.
Result: Cell is moved up. _(Cell should have moved down)_

> FAILURE

---

### Test using **VoiceOver** in **MacOS X**:
1. Add a new cell.
2. Navigate via keyboard through the cell toolbar icons to the **Delete** icon.
3. Follow screen reader prompt to use Ctrl-Option-Space
Result: Success
4. Navigate via keyboard through the markdown toolbar icons to the **Hide/Show Preview** icon.
5. Follow screen reader prompt to use Ctrl-Option-Space
Result: Success

> SUCCESS

---

### Test on Win or Mac **without** screen reader:
1. Add a new code or text cell.
2. Navigate via keyboard through the cell toolbar icons to the **Delete** icon.
3. Hit Enter. 
Result: Nothing happens
4. Navigate via keyboard through the markdown toolbar icons to the **Hide/Show Preview** icon.
5. Hit Enter.
Result: Nothing happens

> _The above are results on Mac or PC when NOT using the screen reader. This may be acceptable; we should check with the a11y team on this._

**Comment by Charles-Gagnon** [comment]
@chlafreniere I'm moving this into the June milestone as part of our cleanup of a11y issues, could you see if this is still relevant and whether it meets the bar for fixing?

**Comment by alanrenmsft** [comment]
bulk closing the usability level issues since we are not planning to work on it anytime soon.
